--- configure	2019-10-19 11:38:44.000000000 -0700
+++ configure	2022-08-17 11:39:16.000000000 -0700
@@ -8467,20 +8467,23 @@
                ;;
        esac
     else
        case `/usr/bin/arch` in
        i386)
                MACOSX_DEFAULT_ARCH="x86_64"
                ;;
        ppc)
                MACOSX_DEFAULT_ARCH="ppc64"
                ;;
+        arm64)
+            MACOSX_DEFAULT_ARCH="arm64"
+            ;;
        *)
                as_fn_error $? "Unexpected output of 'arch' on OSX" "$LINENO" 5
                ;;
        esac

     fi

     LIBTOOL_CRUFT=$LIBTOOL_CRUFT" -lSystem -lSystemStubs -arch_only ${MACOSX_DEFAULT_ARCH}"
     LIBTOOL_CRUFT=$LIBTOOL_CRUFT' -install_name $(PYTHONFRAMEWORKINSTALLDIR)/Versions/$(VERSION)/$(PYTHONFRAMEWORK)'
     LIBTOOL_CRUFT=$LIBTOOL_CRUFT' -compatibility_version $(VERSION) -current_version $(VERSION)';;